### PR TITLE
Interfacing PODIO with Julia Blog GSoC'22

### PR DESCRIPTION
**Pull request by soumilbaldota**
Mid Term Evaluation Blog Post

### CODE REVIEW

**Comment by soumilbaldota** [comment]
@tmadlener I am tagging you to notify you for the midterm evaluation blog post.

**Comment by hegner** [comment]
@soumilbaldota - can you address the spelling mistake as in https://results.pre-commit.ci/run/github/35557163/1658991575.MdR3ZTKdThWu9bMOjwoQZw
Thanks!

**Comment by hegner** [comment]
@soumilbaldota - could you maybe add two sentences about yourself? so that people know who wrote that post?

File: _gsocblogs/2022/blog_PODIO_SoumilBaldota.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+
+In the community bonding period, with the help of Thomas and Benedikt, I setup my development environment and began to work on a prototype for the Interface which described how the generated code would look like. The prototype was created for classes with one to one relations, one to many relations and vector members. Then I made unittests for the prototype and added them to github actions for continuous integration. 
+
+The prototype can be found at the below link
+
+[Prototype](https://github.com/soumilbaldota/PODIO_Julia_Interface_Prototype.git)
+  
+
+After the prototype was created. I added an attribute to store the Julia type of a data member described in C++ to the MemberVariable Class and a function to get the Julia type in [generator_utils.py](https://github.com/AIDASoft/podio/blob/julia/python/generator_utils.py) and added some unittests for the same.
```
Comment (tmadlener) [inline _gsocblogs/2022/blog_PODIO_SoumilBaldota.md:22]: Here you could also link to the pull request that added these changes and add one sentence that explains why this dedicated handling is necessary.

**Review by tmadlener — COMMENTED** [review]
Hi @soumilbaldota,

This is already in quite good shape. I have on correction to the project/title and a few comments on where you could add some more information, resp. highlight your contributions a bit mor.

**Review by hegner — APPROVED** [review]

**Pull request merged**